package.json

```diff
@@ -37,9 +37,11 @@
     "@18f/identity-normalize-yaml": "^1.0.0",
     "@axe-core/puppeteer": "^4.5.1",
     "@babel/eslint-plugin": "^7.14.5",
+    "@babel/preset-react": "^7.22.5",
     "@types/jest": "^29.2.6",
     "@typescript-eslint/eslint-plugin": "^5.48.2",
     "@typescript-eslint/parser": "^5.48.2",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
```

Comment:
Is it necessary to have this dependency? `@babel/preset-react` [includes the JSX transformation plugin](https://babeljs.io/docs/babel-preset-react#:~:text=This%20preset%20always%20includes%20the%20following%20plugins%3A) (the one published under the `@babel/` namespace), so it wouldn't seem to be necessary to install separately?

Reply:
Good call out. I started down a path for one solution and went a different direction. It is not needed.  Resolved with commit 3e46d8cf3a563b33357b66e5af51bec4869ac96c